dotcom-rendering/src/web/components/DynamicFast.tsx

```diff
@@ -80,6 +80,7 @@ export const DynamicFast = ({ trails }: Props) => {
 						starRating={secondary.starRating}
 						branding={secondary.branding}
 						supportingContent={secondary.supportingContent}
+						imagePositionOnMobile="left"
```

Comment:
Now that we need to specify this value I am explicitly adding it in lots of places